### ISSUE TITLE
I try to run cmake 3.14.0 with visual studio 2017 and GDAL224 ,wrong

### ISSUE THREAD

**Issue by laobel** [geo-data/cesium-terrain-builder#82]
严重性	代码	说明	项目	文件	行	禁止显示状态
错误	LNK1120	1 个无法解析的外部命令	ctb	D:\cesium-terrain-builder-master-quantized-mesh\build\src\Debug\ctb.dll	1	

严重性	代码	说明	项目	文件	行	禁止显示状态
错误	LNK2019	无法解析的外部符号 "class GDALDataset * __cdecl GDALCreateOverviewDataset(class GDALDataset *,int,int)" (?GDALCreateOverviewDataset@@YAPAVGDALDataset@@PAV1@HH@Z)，该符号在函数 "void * __cdecl getOverviewDataset(void *,int (__cdecl*)(void *,int,int,double *,double *,double *,int *),void *)" (?getOverviewDataset@@YAPAXPAXP6AH0HHPAN11PAH@Z0@Z) 中被引用	ctb	D:\cesium-terrain-builder-master-quantized-mesh\build\src\GDALTiler.obj	1

**Comment by ahuarte47** [comment]
Related https://github.com/geo-data/cesium-terrain-builder/issues/15#issuecomment-375121426 ?

**Comment by laobel** [comment]
Hi @ahuarte47  I“m try [#15 (comment)](https://github.com/geo-data/cesium-terrain-builder/issues/15#issuecomment-375121426) ?，but the problem still exists.

**Comment by laobel** [comment]
Hi @ahuarte47 ，What GDAL version do you use? x64 or x86？

**Comment by ahuarte47** [comment]
I build CTB with GDAL 2.3.x x64 in Windows

**Comment by laobel** [comment]
Oh,,Please send me the compiled GDAL 2.3.x. Thanks!

**Comment by ahuarte47** [comment]
You can download it from http://www.gisinternals.com

**Comment by laobel** [comment]
I download GDAL 2.3.3 from http://www.gisinternals.com,but errors still exists.

Severity	Code	Description	Project	File	Line	Suppression State
Error	LNK2019	unresolved external symbol "class GDALDataset * __cdecl GDALCreateOverviewDataset(class GDALDataset *,int,int)" (?GDALCreateOverviewDataset@@YAPEAVGDALDataset@@PEAV1@HH@Z) referenced in function "protected: virtual class ctb::GDALTile * __cdecl ctb::GDALTiler::createRasterTile(class GDALDataset *,double (&)[6])const " (?createRasterTile@GDALTiler@ctb@@MEBAPEAVGDALTile@2@PEAVGDALDataset@@AEAY05N@Z)	ctb	D:\cesium-terrain-builder-master-quantized-mesh\build\src\GDALTiler.obj	1

**Comment by ahuarte47** [comment]
Did you try to add to your project the [gdaloverviewdataset.cpp](https://svn.osgeo.org/gdal/trunk/gdal/gcore/gdaloverviewdataset.cpp) file? It was my [workaround](https://github.com/geo-data/cesium-terrain-builder/issues/15#issuecomment-376335635).

**Comment by laobel** [comment]
Okay, thank you for your help.After I added to my project the gdaloverviewdataset.cpp file.Fortunately, the build succeeded.

Thanks!

**Comment by laobel** [comment]
Hi @ahuarte47, I use ctb to create terrain tiles, the format is quantized-mesh. There is an error when loaded with Cesium. Why?
![loadterrainerror](https://user-images.githubusercontent.com/17061454/52755555-55aa3d00-3039-11e9-9c75-0291d5634861.jpg)

**Comment by ahuarte47** [comment]
Do you have defined the `Content-Encoding: gzip` content negotiation header?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding

**Comment by laobel** [comment]
YES。
![qq 20190214095552](https://user-images.githubusercontent.com/17061454/52757168-c6a02380-303e-11e9-93bf-1c6606086a07.jpg)

**Comment by laobel** [comment]
The terrain file is too small,the biggest is 2kb.. Is it normal?
![qq 20190214095919](https://user-images.githubusercontent.com/17061454/52757316-3a423080-303f-11e9-8e0f-30e6df65e9ef.jpg)

**Comment by laobel** [comment]
Request directly in the chrome browser and report the following error.
![qq 20190214100531](https://user-images.githubusercontent.com/17061454/52757657-6f9b4e00-3040-11e9-89a2-f7b3c31f7baa.jpg)

**Comment by ahuarte47** [comment]
Yes, my tiles are very small too. I attach a fragment of my tile set, from 0 to 6th level - [qm.zip](https://github.com/geo-data/cesium-terrain-builder/files/2867301/qm.zip) . It is located in N42°49'0.73" - O1°38'35.63"

I do not know any idea about your error, I'm sorry.

**Comment by laobel** [comment]
Thank you all the same.I try other ideas。

**Comment by laobel** [comment]
Hi @ahuarte47 How do I create quantized-mesh terrain with multiple tif files?Must  merge multiple tif files  into a single file?

**Comment by ahuarte47** [comment]
You could use [gdalbuildvrt](https://www.gdal.org/gdalbuildvrt.html) for creating a virtual raster file without really to save a new big tif.

**Comment by laobel** [comment]
@ahuarte47 How do I create quantized-mesh terrain with water mask by master-quantized-mesh branch。thanks!

**Comment by ahuarte47** [comment]
Sorry, CTB tool does not support the water mask creation for any output format. You can read it in the [limitations-TODO](https://github.com/geo-data/cesium-terrain-builder#limitations-and-todo) section in Readme page.
